### PR TITLE
Adds 10 new drinks, some food reagents and goofconomizes champagne

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request

This PR adds 8 new alcoholic drinks, 2 new nonalcoholic drinks and 3 new food reagents. It also integrates champagne into the goofconomy as a luxury item rather than just being dispenser swill.

## Why It's Good For The Game

This PR adds more variation to bartender play by introducing advanced drinks that require somewhat exotic reagents and often help from other departments.

## Changelog
:cl:
add: 8 new alcoholic drinks.
add: 2 new nonalcoholic drinks.
add: Packs of a novel artificial sweetener have been added to the kitchen vendor.
add: Bottles of trappist beer are now available at a premium price in the booze-o-mat.
add: Juicing parsnips now yields parsnip juice.
add; Maintenance peaches.
del: Removed champagne from the booze dispenser.
balance: Now everyone has to pay 200 credits if they want champagne from the booze-o-mat.
fix: Grape soda now cools you down like other sodas.
imageadd: Added new drink, bottle, vomit and peach can sprites.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (tralezab) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	description = "A turbulent cocktail associated with outlaw hoverbike racing. Not for the faint of heart."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (blessedmulligan) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: googling "Mandela effect 'excist'" doesn't give anything so I'm deciding that this is probably a typo and not me whooshing

**Comment by blessedmulligan** [comment]
make planetcracker give nearby nonhumans badfeels

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: fixed

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Tlaltecuhtli) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: why

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: lol

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (optimumtact) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: lol

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: lol

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: lol

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: lol

**Comment by necromanceranne** [comment]
Aw man, I can't mass produce Wizz Fizz anymore. It's such a pretty sprite too. Bartenders just don't have the cash.

Neither do wizards actually.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (nemvar) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Should be "[M], I am still alive." with a capital I

**Comment by OverlyGeneric** [comment]
I don't know if the orange is laughing at bad code or laughing because it's funny.

I'm glad that the Mandella Effect doesn't pull a Todd Howard.

**Comment by necromanceranne** [comment]
A lot of it is very funny. Old Timer is especially funny. You die from excessive old age.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Mickyan) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: it should randomize between berenstain and berenstein for maximum confusion

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (bgobandit) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: what does "insperable" even mean

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (bgobandit) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: man's past

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (bgobandit) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: >space-American
>flavoured
no u

**Review by harmonyn — COMMENTED** [review]
copy editing shit

**Comment by ExcessiveUseOfCobblestone** [comment]
is this a real pr

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (RaveRadbury) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: "drink was originally was create"

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (XDTM) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: inseparable i suppose

File: code/modules/food_and_drinks/food/snacks_other.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/food_and_drinks/food/snacks_other.dm]: you should use the suggestion feature smh

**Comment by Krysonism** [comment]
> 
> 
> is this a real pr



> 
> 
> is this a real pr

Of course, sweetie. 
This is my first pr.
I am sure we can iron out any rough edges together and make it great.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Intended, but still stupid. I fixed it.

**Comment by Krysonism** [comment]
> 
> 
> Aw man, I can't mass produce Wizz Fizz anymore. It's such a pretty sprite too. Bartenders just don't have the cash.
> 
> Neither do wizards actually.

I think once people start to understand that the bartender actually wants credits I think they will start leaving tips more often. 

As for wizards, if they are competent, they can use the captain's gore covered ID to get all the champagne they want.

**Comment by Krysonism** [comment]
AllDrinks

Here are all the drinks in their glasses. Do note that 3 of them are animated and look better in motion.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: InB4 powergamers start making all of their characters be the minimum possible/allowed age.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Make this deal minor damage to/have a higher boozepwr for nonhuman races.

File: code/modules/reagents/chemistry/reagents/food_reagents.dm
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: A grief chem if I've ever seen one.

I love it.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Maybe make this a 100% chance per tick instead of a 20% chance one so that sec/antags can more easily forcibly age people as a form of punishment?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Tlaltecuhtli) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: make suiciding with it server jump

File: code/game/turfs/turf.dm
Comment (ExcessiveUseOfCobblestone) [inline code/game/turfs/turf.dm]: let's change this a bit, there's no point in having 2 args that are basically both options that can't ever overlap.

`/turf/proc/add_vomit_floor(mob/living/carbon/M, toxvomit = NONE) //NONE already exists as a define`

File: code/game/turfs/turf.dm
Comment (ExcessiveUseOfCobblestone) [inline code/game/turfs/turf.dm]: `if(toxvomit = 2) //DEFINE PLEASE`

File: code/game/turfs/turf.dm
Comment (ExcessiveUseOfCobblestone) [inline code/game/turfs/turf.dm]:
```DM
	else if(vomicon = 1) //define
```

File: code/modules/mob/living/carbon/carbon.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/carbon.dm]:
```DM
				T.add_vomit_floor(src, 2) //use define
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I need a good reason to keep this in, speed boosters are mega cringe.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: there's no downside to it and there's no upside to pick an old age especially now that it just sets you closer to dusting if you have this chem in your system long enough.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I need a good reason to keep this.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: you're better off just checking for holy first then returning ..() since you're doing it regardless.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: is this basically a blink? Also lol getting spaced if you wear anything bluespace.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: cringe

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: not prob 25 == prob 75...

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > 
> 
> I need a good reason to keep this in, speed boosters are mega cringe.

The speed boost is the mildest in the game, ionly 1/4th of the much easier to make and stronger nuka cola. The problem with speed boosters is that they are all too fast, we have not really seen this type of speed booster before. Its high alcohol content limits extended usage.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: This drink is going to be very rare and the teleportation effects only starts if you are drunk enough.

It causes "fits" of teleportation that are hard to powergame since you will get stunned by vomiting after some of the teleports. If you want this to be more friendly to people wearing bluespace gear i can look into changing that.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I am not too attached to this part, i am willing to remove it.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Well, the biggest reason should be the overwhelmingly positive feedback i have received for this drink. Since only you and oranges of like 25 people i have shown this drink to have objected, can you explain to me what the issue is?

The recipe requires corn syrup(corn starch & acid) and therefore takes some effort to make, which will prevent it being made too often.

I can lower the prob of the forced speech or make the ingredients even harder. But without you explaining what you take issue with it is hard for me to address it.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (subject217) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: `Pull requests attempting to add or remove features from said races/species/mutants require prior approval as well.`

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (PKPenguin321) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: cringe

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I want people to be able to enjoy like a half glass without it turning their hair grey. I will consider increasing the rate of age gain though.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I used this approach to easily catch most cases where a player has a tail. If i really need to i could probably make this race neutral.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Fixed

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Removed this

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I changed the teleport channel to anomaly to prevent the spacing issue.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (subject217) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: It's more like "You can probably get away with this but you should ask about it ahead of time". Wouldn't be surprised if maintainers made you dump this entire drink though since it's basically just bait.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > someone tries to PR a drink that makes you go nya and forces a tailwag

yikes

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I thought I btfo'd nuka cola huh

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: sorry this drink needs to go if it is going to have any wagging/nya'ing in it.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: move the conversation up to the other review where I say it's getting removed as it is right now lol

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```dm
var/newsize = pick(0.5, 0.75, 1, 1.50, 2) //you can do this on the same, no need to set a var then set it again without doing anything to it
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: If the resize stays at 1 then this formula will need to be edited.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: this won't if we edit the above properly.

File: code/game/turfs/turf.dm
Comment (Qustinnus) [reply code/game/turfs/turf.dm]: isn't NONE specifically for bitfields

Comment (ExcessiveUseOfCobblestone) [reply code/game/turfs/turf.dm]: I think it's used in bitfields as well, dunno if the define is exclusive to bitfields though.

**Comment by optimumtact** [comment]
I hate that you even make me do this

**Comment by ATH1909** [comment]
@optimumtact Why was this PR closed?

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (PKPenguin321) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: bruh...

**Comment by Krysonism** [comment]
> 
> 
> @optimumtact Why was this PR closed?

I am apparently supposed to make changes within 5 hours at any hour of the day, while maintaining my full time job and sleep schedule. I was asleep when cobby demanded i removed the drink, something i gladly would have done, if i had been awake and aware of his post.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > 
> 
> sorry this drink needs to go if it is going to have any wagging/nya'ing in it.

I am totally willing to remove the drink, as I have been all along. 

All I asked for was a justification. I expected to have to remove it in fact, i did not expect for oranges to close my PR before i could even see this post

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I have an idea of something that might be more unique for this drink instead of the speedboost.

**Comment by RaveRadbury** [comment]
@Krysonism just make the changes and ask for it to be re-opened. It's not like a PR close is a permanent stamp of rejection.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Do you have a suggestion on how best to fix it?  I did not notice any issues with the resizing during play testing. Thanks

**Comment by ExcessiveUseOfCobblestone** [comment]
just reclose if it wasn't due to untimely changes

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I have removed the speed boost and added a more interesting mechanic.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm
Comment (Krysonism) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: The current implementation seems to work flawlessly ingame.

**Comment by Krysonism** [comment]
I think i have made all the changes i needed to make so this should be ready for review.

**Comment by optimumtact** [comment]
cobby get this thing of my tracker

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Comment by Bawhoppen** [comment]
I'm late to the party, but what are maintenance peaches?

**Comment by Krysonism** [comment]
> I'm late to the party, but what are maintenance peaches?
> (and are they infringing on the rightful claim of maintenance oranges???)

Maint peaches spawn in maint, as the name suggest. They were created as a source of a limited availability reagent that you have to scavenge for. Some bad, not good codebases(citadel) have subverted my design by making peach juice renewable thus making blazaam just another drink.

**Comment by Bawhoppen** [comment]
oh okay... I was worried they were a counter-reference to the oranges I added to spawn in maint a long time ago

carry on then I guess

(also very timely reply, TY)

**Pull request merged**
